src/NuGet.Clients/PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -78,8 +78,16 @@ protected override void CreateVersions()
             var installedVersion = installedDependency?.VersionRange?.MinVersion;
 
             var allVersions = _allPackageVersions.OrderByDescending(v => v);
-            var latestPrerelease = allVersions.FirstOrDefault(v => v.IsPrerelease);
-            var latestStableVersion = allVersions.FirstOrDefault(v => !v.IsPrerelease);
+
+            // null, if no version constraint defined in package.config
+            var allowedVersions = _projectVersionRangeDict.Select(kvp => kvp.Value).FirstOrDefault();
```

Comment:
var allowedVersions = _projectVersionRangeDict.Select(kvp => kvp.Value).FirstOrDefault() ?? VersionRange.All

and remove the null check below to simplify this